src/py42/clients/detectionlists/high_risk_employee.py

```diff
@@ -0,0 +1,205 @@
+import json
+
+from py42.clients import BaseClient
+from py42.clients.util import get_all_pages
+from py42.sdk.exceptions import Py42NotFoundError
+
+
+class HighRiskEmployeeClient(BaseClient):
+    """
+    TODO Documentation here
+    """
+
+    # In other modules URI is a single variable, breaking it up to have better control
+    # if we want to refactor version and prefix to a common configuration.
+    _api_version = u"v2"
+    _uri_prefix = u"/svc/api/{0}".format(_api_version)
+
+    def __init__(self, session, user_context):
+        super(HighRiskEmployeeClient, self).__init__(session)
+        self._user_context = user_context
+
+    def _create_user(self, username, tenant_id):
+        resource = u"/user/create"
+        uri = u"{0}{1}".format(self._uri_prefix, resource)
+
+        data = {
+            "tenantId": tenant_id,
+            "userName": username,
+        }
+        return self._session.post(uri, data=json.dumps(data))
+
+    def _add_high_risk_employee(self, tenant_id, user_id):
+
+        data = {"tenantId": tenant_id, "userId": user_id}
+        resource = u"/highriskemployee/add"
+        uri = u"{0}{1}".format(self._uri_prefix, resource)
+        return self._session.post(uri, data=json.dumps(data))
+
+    def add(self, user_id=None, username=None):
```

Comment:
The `userId` is the userUid of the the code42 user (can be seen using `/api/User`, so `username` doesn't have to be a parameter here, we can have this module take a dependency on `userclient` and get the user name for us.

Reply:
Changed to accept only used_id.